samples/snippets/src/test/java/com/example/spanner/SpannerSampleIT.java

```diff
@@ -93,10 +97,45 @@ public static void setUp() throws Exception {
     dbId = DatabaseId.of(options.getProjectId(), instanceId, databaseId);
     // Delete stale test databases that have been created earlier by this test, but not deleted.
     deleteStaleTestDatabases(instanceId, baseDbId);
-    key = String.format("projects/%s/locations/%s/keyRings/%s/cryptoKeys/%s",
-        options.getProjectId(), keyLocation, keyRing, keyName);
+    key =
+        String.format(
+            "projects/%s/locations/%s/keyRings/%s/cryptoKeys/%s",
+            options.getProjectId(), keyLocation, keyRing, keyName);
+
+    /*
+     * Delete stale instances that have been created earlier by this test but not deleted.
+     * Backups needed to be deleted from the instance first, as the instance can only be
+     * deleted once all backups have been deleted.
+     * */
+    deleteStaleTestEncryptedInstances();
+  }
+
+  /**
+   * Deleting all the test instances with name beginning from 'encrypted-test-' and were created
+   * before 24 hours.
+   *
+   * @throws InterruptedException If Thread.sleep() interrupted
+   */
+  static void deleteStaleTestEncryptedInstances() throws InterruptedException {
```

Comment:
Can we change this method to `private`, and change the method name to `deleteStaleEncryptedTestInstances` (in line with the name of the instances) ?
```suggestion
  private static void deleteStaleEncryptedTestInstances() throws InterruptedException {
```